app/src/main/res/values/dimens.xml

```diff
@@ -14,6 +14,42 @@
     <dimen name="margin_6x">48dp</dimen>
     <dimen name="margin_6.5x">52dp</dimen>
     <dimen name="margin_7x">56dp</dimen>
+    <dimen name="margin_7.5x">60dp</dimen>
```

Comment:
woah... please remove all those, we don't need that many, we aren't probably using even half of that!